R/format_value.R

```diff
@@ -120,13 +120,36 @@ check_aligns <- function(algn) {
 #' @return A formatting function which wraps and will apply the specified \code{printf} style format
 #'   string \code{format}.
 #' @seealso \code{\link[base]{sprintf}}
+#' 
+#'  \code{\link[base]{round_fmt}}
+#' 
+#'  \code{\link[base]{list_valid_format_labels}}
 #'
 #' @examples
 #' fmtfun <- sprintf_format("(N=%i")
 #' format_value(100, format = fmtfun)
 #'
 #' fmtfun2 <- sprintf_format("%.4f - %.2f")
 #' format_value(list(12.23456, 2.724))
+#' 
+#' # You can also customise rtables values
+#' library(rtables)
+#' fmtfun3 <- sprintf_format("this range => %.1f")
+#' format_value(100, format = fmtfun3)
+#' 
+#' fmtfun4 <- sprintf_format("sadbasd asa a %.2f asda a %.1f")
+#' format_value(list(12.23456, 2.724), format = fmtfun4)
+#' 
+#' lyt <- basic_table() %>%
+#'   split_cols_by("Species") %>%
+#'   analyze("Sepal.Length", afun = function(x) {
+#'     list(
+#'       "mean (sd)" = rcell(c(mean(x), sd(x)), format = fmtfun4),
+#'       "range" = rcell(diff(range(x)), format = fmtfun3)
+#'     )
+#'   })
```

Comment:
unfortnately you cannot call {rtables} from {formatters}

Reply:
Used `\donotrun{}`

Reply:
donotrun is not enough, we need to take it out

Reply:
@Melkiades That would only leave the `format_value` in place is that what you had in mind? That is, the `lyt` table is not needed for full example? If not, I'll remove 143 to 153, please let me know.

Reply:
Lines completely removed.